config/environments/development.rb

```diff
@@ -52,5 +52,5 @@
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
Commenting this out, in addition to removing some file watcher gems, allowed me to avoid having to restart my server every time I changed a model file.